shell/common/platform_view.cc

```diff
@@ -48,6 +48,10 @@ void PlatformView::DispatchSemanticsAction(int32_t id,
 }
 
 void PlatformView::SetSemanticsEnabled(bool enabled) {
+#if FLUTTER_RUNTIME_MODE == FLUTTER_RUNTIME_MODE_DEBUG
+  FML_DLOG(INFO) << "Debug build. Forcing semantics to enabled.";
```

Comment:
Do we log anything when you run on the iOS simulator? This log may just be noise...

Reply:
We don't, but this log should only show up in --verbose-system-logs